packages/server/database/migrations/20210127153000-removeDupOrgUsers.ts

```diff
@@ -0,0 +1,82 @@
+import {R} from 'rethinkdb-ts'
+import StripeManager from '../../utils/StripeManager'
+import Organization from '../types/Organization'
+export const up = async function(r: R) {
+  const now = new Date()
+  const manager = new StripeManager()
+  try {
+    const duplicateUserIds = await r
+      .table('OrganizationUser')
```

Comment:
this query doesn't work on prod as there are more than 100k rows. we'll need a more efficient query to continue